features/supplier/subscribe_mailing_list.feature

```diff
@@ -13,10 +13,10 @@ Background:
   And I don't see a banner message
 
 Scenario: Successful mailing-list subscription from the home page
-  When I enter 'functional-test-example-email@gov.uk' in the 'email_address' field
+  When I enter 'functional-test-email@user.marketplace.team' in the 'email_address' field
   And I click 'Subscribe'
   Then I am on the 'Digital Marketplace' page
-  And I see a success banner message containing 'You will receive email notifications to functional-test-example-email@gov.uk when applications are opening.'
+  And I see a success banner message containing 'You will receive email notifications to functional-test-email@user.marketplace.team when applications are opening.'
```

Comment:
As per previous comment, this one is Mailchimp rather than Notify, right? Can we use `user.marketplace.team` in the same way here?

Reply:
Well, as far as I'm aware, `@user.marketplace.team` isn't hooked up to anything in exactly the same way `@gov.uk` isn't, so I imagine it will have the same effect.

Reply:
Well, as far as I'm aware, `@user.marketplace.team` isn't hooked up to anything in exactly the same way `@gov.uk` isn't, so I imagine it will have the same effect.

Reply:
I double checked the manual and it says that *any* Mandrill email on preview/staging goes to a sandbox (is that right?) https://alphagov.github.io/digitalmarketplace-manual/emails.html?highlight=emails#id3

Reply:
But Mailchimp doesn't have a sandbox - there are preview/staging mailing lists instead.

Reply:
I ... don't know. Didn't think we still used mandrill itself? Still, this test works with `user.marketplace.team`, and changing it reduces the number of domains we use.